src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Update/Endpoint.cs

```diff
@@ -19,9 +20,12 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
         var user = await userService.GetUserFromJwtAsync(ct);
         var existingMt = await dbContext.ResourceContentVersionMachineTranslations
             .AsTracking()
+            .Include(rcvmt => rcvmt.ResourceContentVersion)
+            .ThenInclude(rcv => rcv.ResourceContent)
             .FirstOrDefaultAsync(x => x.Id == req.Id, ct);
 
-        if (existingMt is null || existingMt.UserId != user.Id)
+        if (existingMt is null || existingMt.ResourceContentVersion.AssignedUserId != user.Id ||
```

Comment:
Should I clean some of this up while I'm at it? maybe do a ValidateXXX method, and an UpdateXXX method for the other changes?

Reply:
I think we can do these checks in the `FirstOrDefaultAsync` and then just check `existingMt is null`.